linera-execution/src/runtime_actor/requests.rs

```diff
@@ -54,6 +54,18 @@ pub enum BaseRequest {
         response_sender: oneshot::Sender<()>,
     },
 
+    /// Requests to read an entry from the key-value store.
```

Comment:
Minor: I think this needs to be updated

```suggestion
    /// Requests to check if a key is present in the key-value store.
```

Reply:
Ok.